Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.css

```diff
@@ -42,16 +42,25 @@
     color: var(--text-color-secondary);
 }
 
-.details-section > .content > .group > .row.font-variation > .value {
+.details-section > .content > .group > .row.font-variation > input[type="number"] {
     grid-column: axis-value;
-    text-align: end;
+    height: max-content;
+    width: 6ch;
+    padding-block: 4px;
+    padding-inline: 0.5ch;
+    margin-block-start: -6px;
+    border: 1px solid var(--border-color);
     font-variant-numeric: tabular-nums;
 }
 
-.details-section > .content > .group > .row.font-variation:has(.warning) > .value {
+.details-section > .content > .group > .row.font-variation:has(.warning) > input[type="number"] {
     margin-inline-end: 8px; /* Clearance so warning icon doesn't overlap the axis value. */
```

Comment:
Should we always ensure this clearance? It seems strange that we would cause the slider and text box to shift when a warning is present. This both may be unsettling if the text field you are typing in shifts, as well as will break the visual rhythm as you look up and down a list of these fields since the boxes and sliders may no longer align.